drizzlepac/haputils/astrometric_utils.py

```diff
@@ -577,15 +572,13 @@ def build_auto_kernel(imgarr, whtarr, fwhm=3.0, threshold=None, source_box=7,
     kern_img[:, -edge:] = 0.0
     kernel_psf = False
 
-    peaks = find_peaks(kern_img, threshold=threshold * 5,
-                       box_size=isolation_size)
+    peaks = photutils.detection.find_peaks(kern_img, threshold=threshold * 5, box_size=isolation_size)
     if peaks is None or (peaks is not None and len(peaks) == 0):
         tmean = threshold.mean() if isinstance(threshold, np.ndarray) else threshold
         if tmean > kern_img.mean():
             kern_stats = sigma_clipped_stats(kern_img)
             threshold = kern_stats[2]
-        peaks = find_peaks(kern_img, threshold=threshold,
-                           box_size=isolation_size)
+        peaks = photutils.detection.find_peaks(kern_img, threshold=threshold, box_size=isolation_size)
```

Comment:
Again, 'photutils.detection' can be removed from this line due to the change in imports at the top of the module.

Reply:
issue resolved by commit [8892775](https://github.com/spacetelescope/drizzlepac/pull/986/commits/8892775dfdbf046c425976b4d1d831517eae30bf)